project/src/main/java/com/google/autograder/data/Database.java

```diff
@@ -40,27 +40,33 @@ public void addAssignment(String name, int totalPoints) {
     this.datastore.put(assignmentEntity);
   }
 
```

Comment:
Good work in here. For this Database class though specifically, like I said with Mahima, I think we should merge some of the older PRs into main before this one. I think it'll be easier to copy these few methods into there then try to merge a whole new Database structure on top of this file.

Reply:
Should I stick this in a separate file then?

Reply:
Let's chat about this, I'll send out a message.